IBAnimatable/PlaceholderDesignable.swift

```diff
@@ -18,7 +18,8 @@ public extension PlaceholderDesignable where Self: UITextField {
   var placeholderText: String? { get { return "" } set {} }
 
   public func configurePlaceholderColor() {
-    if let placeholderColor = placeholderColor, let placeholder = placeholder {
+    let text = placeholderText ?? placeholder
```

Comment:
Sorry, hand on a sec. `var placeholderText: String? { get { return "" } set {} }` will never return `nil`. It just returns empty string. Will it be a problem?